src/main/kotlin/com/sourcegraph/cody/config/notification/AccountSettingChangeContext.kt

```diff
@@ -2,5 +2,8 @@ package com.sourcegraph.cody.config.notification
 
 class AccountSettingChangeContext(
     val serverUrlChanged: Boolean = false,
-    val accessTokenChanged: Boolean = false
-)
+    val accessTokenChanged: Boolean = false,
+    val accountTierChanged: Boolean = false
```

Comment:
It was added mostly for clarity so e.g. in debugger when you will notice `AccountSettingChangeContext` it will be clear why it was emitted. With just two other properties set to false it could look like a bug.